### PR TITLE
Add vnx util stub to write ARP table entry

### PR DESCRIPTION
**Pull request by 108anup**

### CODE REVIEW

File: Notebooks/vnx_utils.py [hunk 1]
```diff
@@ -439,6 +439,44 @@ def readARPTable(self, num_entries=256) -> dict:
 
         return table
 
+    def writeARPEntry(self, mac, ip):
```
Comment (mariodruiz) [inline Notebooks/vnx_utils.py:442]: Please use snake case to be compatible with PEP8, I know that the rest of the code does not use snake case for many methods. I have an action to fix that.

File: Notebooks/vnx_utils.py [hunk 2]
```diff
@@ -439,6 +439,44 @@ def readARPTable(self, num_entries=256) -> dict:
...
+        """
+
+        if not isinstance(mac, str):
+            raise ValueError("MAC address must be a string.")
+        elif not isinstance(ip, str):
+            raise ValueError("IP address must be a string.")
+
+        mac_int = int("0x{}".format(mac.replace(":", "")), 16)
+        little_mac_int = _byteOrderingEndianess(mac_int, 6)
+        mac_msb = little_mac_int // (2 ** 32)
```
Comment (mariodruiz) [inline Notebooks/vnx_utils.py:460]: Right shift by 32 and mask, (little_mac_int >> 32) & 0xFFFFFFFF

File: Notebooks/vnx_utils.py [hunk 3]
```diff
@@ -439,6 +439,44 @@ def readARPTable(self, num_entries=256) -> dict:
...
+        mac_lsb = little_mac_int % (2 ** 32)
```
Comment (mariodruiz) [inline Notebooks/vnx_utils.py:461]: Mask, little_mac_int & 0xFFFFFFFF

File: Notebooks/vnx_utils.py [hunk 4]
```diff
@@ -439,6 +439,44 @@ def readARPTable(self, num_entries=256) -> dict:
...
         return table
 
+    def writeARPEntry(self, mac, ip):
+        """Add an entry to the ARP table
+
+        Parameters
+        ----------
+        mac: str
+            MAC address in the format XX:XX:XX:XX:XX:XX
+        ip: str
```
Comment (mariodruiz) [inline Notebooks/vnx_utils.py:449]: As the IP has to be in the same subnet /24. What do you think of passing this as an 8-bit index? The in can be completed with the 3 MSB bytes of the main IP

**Review by mariodruiz — CHANGES_REQUESTED** [review]

**Comment by mariodruiz** [comment]
Part of #56

**Comment by 108anup** [comment]
LMK if you want me to squash commits.

**Pull request merged**